js_modules/dagit/packages/core/src/runs/LogsToolbar.tsx

```diff
@@ -95,6 +90,11 @@ export const LogsToolbar: React.FC<ILogsToolbarProps> = (props) => {
     [onSetLogType, setInitialComputeLogType],
   );
 
+  const activeItems = React.useMemo(
+    () => new Set([logType === LogType.structured ? logType : initialComputeLogType]),
```

Comment:
This is a small bug fix that prevented the "stderr" tab from showing an active item in the structured vs. raw picker